docs/source/landing_page.rst

```diff
@@ -0,0 +1,53 @@
+.. _landing_page:
+
+Thoth's landing page
+--------------------
+
+Here you can find all the relevant information if you want to get in touch with
+us or get involved.
+
+.. note::
+
+  If you do not find required information, feel free to contact us in
+  `Google Chat room <https://chat.google.com/room/AAAAVjnVXFk>`__.
```

Comment:
This one is tricky, I figured out that we need to invite people on the outside from the inside, so they can not actively join the channel :/ Maybe we need them to open an issue, so that we can manually invite them?

I know, its odd...

Reply:
we need to update it also here: https://github.com/thoth-station/core#interact-with-thoth-team

Reply:
Adjusted, PTAL.